app/assets/stylesheets/application/slideout.sass

```diff
@@ -0,0 +1,47 @@
+@import application
+@import application/sizes
+
+.slideout
+	height: 100%
+	.trigger
+		position: absolute
+		right: 0px
+		padding: 5px 10px 10px 10px
+		border-radius: 0px 0px 0px 10px
+		background: black
+		color: white
+		height: $panel_height
+		&:hover
+			cursor: pointer
+			text-decoration: underline
+
+	.slideout_body
+		@extend .col-md-3
+		padding: 0px !important
+		height: inherit
+		&.offscreen	
+			width: 0%
+			padding: 0px
+			overflow: hidden
+			height: 0px
+
+	.projects_preview_header
+		height: $panel_height
+		background-color: lightpink
+		font-size: 21px
+		font-weight: 300
+		text-align: center
+
+	.projects_preview_body
+		overflow: scroll
+		height: calc(100% - #{$panel_height})
+
+.body_panel, .slideout_body
+	-webkit-transition: width 0.3s ease, margin 0.3s ease
+	-moz-transition: width 0.3s ease, margin 0.3s ease
+	-o-transition: width 0.3s ease, margin 0.3s ease
+	transition: width 0.3s ease, margin 0.3s ease
```

Comment:
generally I like to use sass to help me DRY code like this up. You can make a `mixins.scss` file with things like `transition` and have the `transition` mixin duplicate the `transition, -o-transition, -moz-transition, etc` code for you